lib/logging/third_party_transaction.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module Logging
+  module ThirdPartyTransaction
```

Comment:
This file was moved from `ThirdPartyTransactionLogging`, so unfortunately some git history was lost. Despite using `git mv`, the indentation of the new name spacing breaks git's ability to recognize this automatically as the same file.  There is probalby a way to do it manually, but I'm not sure it's worth the time.

Notable changes to this file are
1. location and namespace
2. addition of appropriate return value.  Previously the overwritten methods return value was lost
3. additional logging of 'passed args', which will probably be semi-useless stuff like <MyObject #23423498> most of the time, but imho is still nice to see.  This logging is added to both begin / complete methods